src/app/jscaip/Coord.ts

```diff
@@ -5,6 +5,10 @@ import { ComparableObject } from '../utils/Comparable';
 import { MGPFallible } from '../utils/MGPFallible';
 import { Encoder, NumberEncoder } from '../utils/Encoder';
 
+export class CoordFailure {
+    public static readonly OUT_OF_RANGE: (coord: Coord) => string = (coord: Coord) => $localize`Coord out of range, invalid move! ` + coord.toString();
```

Comment:
à ne pas localiser ? Enfin soit on s'applique sur le message et on localise, soit ni l'un ni l'autre :D 
Coord -> Coordinate
Plutôt que de juste appender la valeur après le message, l'intégrer plus correctement à celui-ci 
On est dans coord, pas dans move, donc pas de raison de parler de move ici

Donc -> "The coordinate (-1, 2) is not on the board" / "La coordonnée (-1, 2) est hors du plateau" ?